screeps-game-api/src/constants.rs

```diff
@@ -294,17 +299,18 @@ pub mod look {
     };
 
     typesafe_look_constants! {
-        CREEPS, Look::Creeps, Creep;
-        ENERGY, Look::Energy, Resource;
-        RESOURCES, Look::Resources, Resource;
-        SOURCES, Look::Sources, Source;
-        MINERALS, Look::Minerals, Mineral;
-        STRUCTURES, Look::Structures, Structure;
-        FLAGS, Look::Flags, Flag;
-        CONSTRUCTION_SITES, Look::ConstructionSites, ConstructionSite;
-        NUKES, Look::Nukes, Nuke;
-        TERRAIN, Look::Terrain, Terrain;
-        TOMBSTONES, Look::Tombstones, Tombstone;
+        CREEPS, Look::Creeps, Creep,  ::objects::IntoExpectedType::into_expected_type;
+        ENERGY, Look::Energy, Resource,  ::objects::IntoExpectedType::into_expected_type;
+        RESOURCES, Look::Resources, Resource,  ::objects::IntoExpectedType::into_expected_type;
+        SOURCES, Look::Sources, Source,  ::objects::IntoExpectedType::into_expected_type;
+        MINERALS, Look::Minerals, Mineral,  ::objects::IntoExpectedType::into_expected_type;
+        STRUCTURES, Look::Structures, Structure,  ::objects::IntoExpectedType::into_expected_type;
+        FLAGS, Look::Flags, Flag,  ::objects::IntoExpectedType::into_expected_type;
+        CONSTRUCTION_SITES, Look::ConstructionSites, ConstructionSite,
+             ::objects::IntoExpectedType::into_expected_type;
+        NUKES, Look::Nukes, Nuke,  ::objects::IntoExpectedType::into_expected_type;
+        TERRAIN, Look::Terrain, Terrain,  ::stdweb::unstable::TryInto::try_into;
+        TOMBSTONES, Look::Tombstones, Tombstone,  ::objects::IntoExpectedType::into_expected_type;
```

Comment:
True! The problem is that `::objects::IntoExpectedType::into_expected_type` and `::stdweb::unstable::TryInto::try_into` are pretty much the shortest for I can make them while still being able to differentiate between the two.

I suppose these could import `IntoExpectedType` and `TryInto` and leave out the full expanded form.

Reply:
I've imported them in but I don't think there is more that I can do to make this nicer since we do need to differentiate between `IntoExpectedType` and `TryInto` for each one.

Reply:
You couldn't have made two arms to the macro? 
In the end, it doesn't matter, it gets expanded to the same code, it was just funny to see all the trouble you gave yourself 😛